code/iaas/accounts/src/main/resources/META-INF/cattle/accounts/spring-accounts-context.xml

```diff
@@ -149,10 +153,29 @@
             </list>
         </property>
     </bean>
-    
+
     <bean id="Project" class="io.cattle.platform.api.schema.ObjectBasedSubSchemaFactory" >
         <property name="id" value="project" />
-        <property name="schemaFactory" ref="User" />
+        <property name="schemaFactory" ref="AdminSchema" />
+        <property name="postProcessors">
+            <list>
+                <bean class="io.cattle.platform.schema.processor.JsonFileOverlayPostProcessor" >
+                    <property name="path" value="schema/project" />
+                </bean>
+                <bean class="io.cattle.platform.schema.processor.AuthOverlayPostProcessor" >
+                    <property name="resources" >
+                        <bean class="io.cattle.platform.spring.resource.SpringUrlListFactory" >
+                            <property name="resources">
+                                <list>
+                                    <value>classpath:schema/user/user-auth.json</value>
```

Comment:
Just for my understanding, are we temporarily pointing to 'user-auth.json' and 'user-auth.json.d/**/*.json' here as a holdover until the next PR.  Is the next PR going to introduce a project-auth.json (that is basically the same as the current user-auth.json) and then the new user-auth.json will become a lot more limited?